deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -417,7 +424,14 @@ class WebxdcViewController: WebViewViewController {
     private func loadHtml() {
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
             guard let self else { return }
-            let url = URL(string: "\(self.INTERNALSCHEMA)://acc\(self.dcContext.id)-msg\(self.messageId).localhost/index.html")
+
+            let file = if let href {
+                href
+            } else {
+                "index.html"
+            }
```

Comment:
This is the same as 

```swift
let file = href ?? "index.html"
```

Reply:
yeah, soo much switching of programming languages these days :) thanks for the pointer, i simplified that futher